src/signed_data.rs

```diff
@@ -352,6 +372,36 @@ const RSA_PKCS1_SHARED: PublicKeyAlgorithmSharedInfo =
     allowed_signature_alg_parameters: &[&[], &[0x05, 0x00]], // Optional NULL.
 };
 
+const RSA_PSS_SHA256: PublicKeyAlgorithm = PublicKeyAlgorithm {
+    shared: &PublicKeyAlgorithmSharedInfo {
+      spki_algorithm_oid: &oid_1_2_840_113549![1, 1, 1],
```

Comment:
Let's create a name for this, `RSA_ENCRYPTION_OID`, and update the reference to it to use the name. That makes it clear that this isn't the RSA-PSS OID.